prboom2/src/SDL/i_system.c

```diff
@@ -513,3 +513,8 @@ char* I_FindDeh(const char* wfname)
 
   return I_FindFile(wfname, ".deh");
 }
+
+char* I_FindZip(const char* wfname)
```

Comment:
Please add an `I_RequireZip` as well so we have consistent interface options.

Reply:
Added in [640f573](https://github.com/kraflab/dsda-doom/pull/292/commits/640f573eb115a4b37863151e1b45dfb7cec4d62f).